pandas/tests/extension/base/methods.py

```diff
@@ -323,7 +323,7 @@ def test_repeat(self, data, repeats, as_series, use_numpy):
         self.assert_equal(result, expected)
 
     @pytest.mark.parametrize('repeats, kwargs, error, msg', [
-        (2, dict(axis=1), ValueError, "'axis"),
+        (2, dict(axis=1), ValueError, "'?axis"),
```

Comment:
We does this change?

Reply:
The test seemed to raise with right error message, but without the quote. I'll revert and see if the CI fails again.